### ISSUE TITLE
My colors changed after the update

### ISSUE THREAD

**Issue by wordyallen** [silvestreh/atom-material-syntax-light#9]
Is this a temporary thing?
![](https://i.gyazo.com/2bc178e6b1dcdb3e2f25b60a712827ec.png)

**Comment by silvestreh** [comment]
I "synced" it with the Sublime version, so yeah, some things might have changed. You can always install a previous version through APM:

```shell
apm install atom-material-syntax-light@0.4.5
```

**Comment by wordyallen** [comment]
cool. thanks.

**Comment by bradstewart** [comment]
For anyone else wondering why JS keywords  were suddenly italicized, this was it.

**Comment by silvestreh** [comment]
Yeah, I'm starting to actually write the themes from scratch rather than just converting from sublime's version to avoid these kind of issues in the future.

The converted theme is pretty hard to maintain because the auto-generated CSS code isn't too good.

Stay tuned for future updates on all variants of the syntax theme 😄

**Comment by lednhatkhanh** [comment]
Thank you @silvestreh Waiting for forever for the update!
